### DISCUSSION TITLE
How to fix make

### DISCUSSION THREAD

**Discussion by aiblockchain1** [Cyfrin/foundry-full-course-cu#1871] [Q&A]
Tried to use make comments error ???
make: *** No rule to make target 'deploy'.  Stop.
Pls help.

**Comment by EngrPips** [comment]
Hello @aiblockchain1, Please check this link `https://github.com/Cyfrin/foundry-full-course-f23/blob/main/how-to-ask-a-question.md` to know how to format your question properly.
